samples/python/kernel_http_server/local.settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "IsEncrypted": false,
+  "Host": {
+    "CORS": "*"
+  },
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "python",
+    "AzureWebJobsStorage": "",
```

Comment:
lines 8-10 are not utilized for the C# azure function. They are also causing errors during startup on my local machine. I believe they can be deleted safely.

Reply:
I don't think AzureWebJobsStorage and AzureWebJobs.KernelHttpServer.Disabled are necessary, but I do need the line `"AzureWebJobsFeatureFlags": "EnableWorkerIndexing"` or it won't find the functions